examples/knative/python/kn-py-vm-attr/README.md

```diff
@@ -0,0 +1,166 @@
+# kn-vm-creation-attr-fn
+
+> This function is greatly inspired by the content from [embano1/kn-echo](https://github.com/embano1/kn-echo) and
+> [lrivallain/openfaas-fn/.../vm-creation-attr-fn](https://github.com/lrivallain/openfaas-fn/tree/master/vm-creation-attr-fn).
+
+A sample python function with `Flask` REST API running in Knative to apply _custom attributes_
+ to a VM object based on an event submitted by the [VEBA](https://vmweventbroker.io/) 
+ (VMware Event Broker) services:
+
+* `event-creation_date`: To store the creation date
+* `event-last_poweredon`: To store the last powered on date
+* `event-owner`: To store the user that created the VM
+
+![Custom attributes](./screenshots/custom_attributes.png)
+
+> **Note:** CloudEvents using structured or binary mode are supported.
+
+
+# Step 1 - Build with `pack`
```

Comment:
Might want to add a note that if the user does not want to modify anything in the function example, they could straight jump to deploy